floors/enforce.go

```diff
@@ -0,0 +1,45 @@
+package floors
+
+import (
+	"github.com/mxmCherry/openrtb/v16/openrtb2"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func ShouldEnforce(bidRequest *openrtb2.BidRequest, floorExt *openrtb_ext.PriceFloorRules, configEnforceRate int, f func(int) int) bool {
+
+	if floorExt != nil && floorExt.Skipped != nil && *floorExt.Skipped {
+		var floorInRequest bool
+		for i := range bidRequest.Imp {
+			if bidRequest.Imp[i].BidFloor > 0 {
+				floorInRequest = true
+				break
+			}
+		}
+		if !floorInRequest {
+			return floorInRequest
+		}
```

Comment:
Consider extracting to a method, like:

```
func requestHasFloors(bidRequest *openrtb2.BidRequest) bool {
	for i := range bidRequest.Imp {
		if bidRequest.Imp[i].BidFloor > 0 {
			return true
		}
	}
	
	return false
}
```

Then you can use return statements, describe the functionality via the function name, and make it easier to isolate for testing.

Reply:
Thank you for suggestion, 
Even I thought of adding same function 😄